pyrsistent/__init__.pyi

```diff
@@ -154,16 +155,16 @@ class PRecord(PMap[AnyStr, Any]):
     def discard(self: T_PRecord, key: KT) -> T_PRecord: ...
     def remove(self: T_PRecord, key: KT) -> T_PRecord: ...
 
-    def serialize(self, format: Optional[Any] = ...) -> MutableMapping: ...
+    def serialize(self, format: Optional[Any] = ...) -> Dict[str, Any]: ...
```

Comment:
Should the keys be `AnyStr` rather than `str`?

Reply:
I suppose technically yes, although it's honestly weird to me that `PRecord` is typed with `AnyStr` in the first place because I'm pretty sure python only allows `str` values for the names of fields/keyword arguments so that would naturally restrict the valid record fields...